language_detector/main.py

```diff
@@ -1,8 +1,22 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
 
 def detect_language(text, languages):
     """Returns the detected language of given text."""
     # implement your solution here
-    pass
+    #def languages_common_words:
+    lang=""
+    prevcount=0
+    for language in languages:
+        count=0
+        x=language.get('common_words')
+        for word in x:
+            if word in text:
```

Comment:
Just a note here, this could result in false positives. Using `in` against a string is a substring match, this means that `'cd' in 'abcdefg'` would return True. The way around this would be to call `.split()` on `text` first as this would split the string into a list, using whitespace (or a specific string if given to split as a parameter) as the delimiter. Example: `'Split me baby one more time.'.split()` would return the following list: `['Split', 'me', 'baby', 'one', 'more', 'time.']`

Also, you aren't taking case sensitivity into account. The `common_words` list is all lower case, and thus won't match any words with upper case letters in them. Using `.lower()` on a string will return the string converted to all lower case. Eg.: `"My name is Bob.".lower()` will return `"my name is bob."`.